test/functional/p2p_blockfilters.py

```diff
@@ -50,13 +66,23 @@ def set_test_params(self):
             ["-blockfilterindex"],
         ]
 
+    def skip_test_if_missing_module(self):
```

Comment:
please remove `skip_if_no_wallet` and pick this commit ab49c5d7d089d2f0c59715f48731016a731ed75c

It removes duplicated code for asset-lock creation and removes dependency on wallet for this functional test.

Review carefully `ab49c5d7d089d2f0c59715f48731016a731ed75c` to be sure that it still tests all cases as you planned

Reply:
?